src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -73,10 +79,19 @@ public void onCommand(Member sender, CommandContext context) {
         builder.addField("User ID", userForInfo.getId(), true);
         builder.addField("Mutual Servers", String.valueOf(userForInfo.getMutualGuilds().size()), true);
 
-
+        if (member != null) {
+            PermissionsManager permissionsManager = new PermissionsManager();
```

Comment:
Don't make a new permission manager. you can get it via the CascadeBot instance

Reply:
Which is soon to be Cascade, if the decision in a meeting is passed. I'd say leave it? @weeryan17

Reply:
> Don't make a new permission manager. you can get it via the CascadeBot instance

Using static stuff on non-static stuff? Bad idea 
(or I'm just dumb)

Reply:
That's not the point here. if we do change the class name it will auto refactor this. Ether way making a new manager will break things so grab it from whatever we're calling the main class. Also we haven't decided on it yet so it might not be renamed, but again rename doesn't matter

Reply:
> 
> Using static stuff on non-static stuff? Bad idea
> (or I'm just dumb)

Depends on the situation. using instance objects is actually very common as it's a great way to store managers and such.

Reply:
And in this case not using the instance can break stuff.

Reply:
Suggestion?

Reply:
As in this case it will not compile

> Suggestion?

In _this_ case it will *not* compile

Reply:
refer to my commit to your banch